connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceTask.java

```diff
@@ -266,7 +262,29 @@ private Map<TopicPartition, Long> loadOffsets(Set<TopicPartition> topicPartition
     private Long loadOffset(TopicPartition topicPartition) {
         Map<String, Object> wrappedPartition = MirrorUtils.wrapPartition(topicPartition, sourceClusterAlias);
         Map<String, Object> wrappedOffset = context.offsetStorageReader().offset(wrappedPartition);
-        return MirrorUtils.unwrapOffset(wrappedOffset) + 1;
+        return MirrorUtils.unwrapOffset(wrappedOffset);
+    }
+
+    // visible for testing
+    void initializeConsumer(Set<TopicPartition> taskTopicPartitions) {
+        Map<TopicPartition, Long> topicPartitionOffsets = loadOffsets(taskTopicPartitions);
+        consumer.assign(topicPartitionOffsets.keySet());
+        log.info("Starting with {} previously uncommitted partitions.", topicPartitionOffsets.values().stream()
+                .filter(this::isUncommitted).count());
+
+        log.trace("Seeking offsets: {}", topicPartitionOffsets.entrySet().stream()
```

Comment:
`topicPartitionOffsets` here maybe not a correct offset for committed offsets(actually needs to plus 1), right? So the log here may be misleading. I think the line 285 is correct. Maybe the two places can retain one.

Reply:
Good point. Considering they're both at `TRACE` level, I think it's fine to remove this one.